server/pom.xml

```diff
@@ -2083,6 +2083,11 @@
           </dependency>
         </dependencies>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+      </plugin>
```

Comment:
We should just add the plugin to the plugins section in parent, then we won't have to repeat it in api, common, client and server.